### PR TITLE
fix: #189 - Registration documentation

### LINKED ISSUE #189: Document the files in mifopay/registration package
**Summary:**

Document the files in mifopay/registration package: https://github.com/openMF/mobile-wallet/tree/master/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration

### PR DESCRIPTION
**Pull request by theamritanair**
Fixes #189 
- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by theamritanair** [comment]
@ankurs287  @luckyman20 please review this

**Comment by Bhavnaharitsa** [comment]
@luckyman20 @theamritanair Are these necessary in the first place?

**Comment by luckyman20** [comment]
@Bhavnaharitsa Yes, these are necessary for maintaining the quality of code. These helps new contributors to get accustomed to codebase easily.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/MobileVerificationActivity.java [hunk 1]
```diff
@@ -105,12 +116,19 @@ public void onRequestOtpSuccess() {
...
+     */
     @Override
     public void onRequestOtpFailed(String s) {
         hideProgressDialog();
         showToast(s);
     }
 
+    /**
+     * This function is implemented when Next is clicked.
+     */
```
Comment (naman14) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/MobileVerificationActivity.java]: The function signature is already verbal enough to signify 'Next is clicked'
Instead the documentation should be what does this do (it's hiding the keyboard and why the delay)

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/SignupMethod.java [hunk 2]
```diff
@@ -64,6 +72,9 @@ public void onMerchantClicked() {
         }
     }
 
+    /**
+     * This function signs up when the customer is clicked.
```
Comment (naman14) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/SignupMethod.java]: This comment is too confusing . What is customer? What does sign up means?

**Comment by rachittt** [comment]
Closing this because of inactivity.

**Pull request closed without merging**